<launch>
	<include file="$(find turtlebot_bringup)/launch/minimal.launch"/>
	<include file="$(find turtlebot_custom)/launch/gmapping_lidar.launch"/>
	<include file="$(find turtlebot_custom)/launch/lidar.launch"/>
    <include file="$(find turtlebot_custom)/launch/rviz.launch"/>
  <node name="respeaker_node" pkg="respeaker_ros"  type="respeaker_node.py"/>
  <node name="static_transformer" pkg="tf" type="static_transform_publisher"
        args="0 0 0 0 0 0 map respeaker_base 100"/>

  <node pkg="restaurant" name="call_ducker" type="call_ducker.py" output="screen"/>

  <node pkg="sound_system" name="speak" type="speak.py" output="screen"/>
  <node pkg="sound_system" name="hotword" type="hotword_detector.py" output="screen"/>

  <group ns="move">
     <node pkg="move" name="velocity" respawn="false" type="velocity"/>
     <node pkg="move" name="amount" respawn="false" type="amount"/>
  </group>

  <include file="$(find realsense2_camera)/launch/rs_rgbd.launch"/>

  <node name="pose_estimator" pkg="tfpose_ros" type="broadcaster_ros.py" output="screen" required="true">
      <env name="CUDA_VISIBLE_DEVICES" value="0" />
      <param name="model" value="mobilenet_v2_large" />
      <param name="resolution" value="640x480" />
  </node>
  <node name="realsense" pkg="tfpose_ros" type="tf_pose_realsense_once" output="screen" required="true"/>


</launch>
